src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,61 @@
-export const Tabs = () => {};
+export type Tab = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+type TabsProps = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: TabsProps) => {
+  const isLinkActive = (tabId: string) => {
+    const isAnySelected = tabs.some(tab => tab.id === selectedTabId);
+
+    if (isAnySelected && tabId === selectedTabId) {
+      return 'is-active';
+    }
+
+    if (!isAnySelected && tabId === tabs[0].id) {
+      return 'is-active';
+    }
+
+    return undefined;
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            return (
```

Comment:
suggestion: you can reduce this code by removing this return and curly brackets and using just `( )`